.travis.yml

```diff
@@ -0,0 +1,43 @@
+sudo: false
+language: python
+python:
+  - 3.7-dev
+  - pypy3
+cache: pip
+
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27-django19
+    - python: 2.7
+      env: TOXENV=py27-django110
+    - python: 3.5
+      env: TOXENV=py35-django19
+    - python: 3.5
+      env: TOXENV=py35-django110
+    - python: 3.6
+      env: TOXENV=py36-django19
+    - python: 3.6
+      env: TOXENV=py36-django110
+    - python: 3.6
+      env: TOXENV=py36-django111
+    - python: pypy
+      env: TOXENV=pypy-django111
```

Comment:
Is there any actual reason we test against PyPy?

Reply:
maybe? i just used the same envs we use for other packages.